xAPI.md

```diff
@@ -260,49 +260,30 @@ learning that SCORM could not enable.
 	*	[Appendix G: Cross Domain Request Example](#AppendixG)  
 
 <a name="partone"/>
-#Part One: About The Experience API
+#PART ONE: LEARNING EXPERIENCES
 
+<a name="Introduction"/>
```

Comment:
Thanks - good point - I will change.  How would you feel about a convention that uses the full name of the Section title in the tag - lower case and using underscores rather than dashes?  I found that doing this made it much easier to navigate the document in my text editor as I could click on part_one_introduction and get the whole phrase where as if there were dashes, the editor would just select whatever word I was on.